addons/sale/controllers/portal.py

```diff
@@ -35,6 +35,10 @@ def _prepare_home_portal_values(self, counters):
 
         return values
 
+    def _get_additional_order_page_values(self, sale_order=None):
```

Comment:
Is there a need to support the case where the SO is not provided, it should always exists where you're calling this method :thinking: 

Edit: In fact, the code of the route is confusing but `_document_check_access` always return a valid record or raises, so you can consider `order_sudo` to always be available :)

Reply:
Also, do you mind moving this method directly on the SO model ?
I believe it might be easier for inheritance if we provide a method like `sale_order._get_portal_rendering_values()`

Reply:
Move to SO model is done. I think I had a brain fart and should have thought to do that in the first place, thanks!